code/datums/datum.dm

```diff
@@ -4,7 +4,7 @@
 	var/list/datum_components //for /datum/components
 	var/list/status_traits
 	var/list/comp_lookup //it used to be for looking up components which had registered a signal but now anything can register
-	var/list/signal_procs
+	var/list/list/datum/callback/signal_procs

```

Comment:
Was it intentional to have two /list's?

Reply:
Yes, `signal_procs` is a list of lists of callbacks. The first index is the recipient of the signal, and the second index is the signal type. See `/datum/proc/RegisterSignal`.

Reply:
cool ❤️